packages/playground/src/components/applications/ApplicationCards.vue

```diff
@@ -1,10 +1,23 @@
 <template>
   <v-row>
-    <v-col v-for="card in cards" :key="card.title" cols="12" sm="12" md="12" lg="6" :xl="cards.length > 3 ? 4 : 6">
+    <v-col
+      v-for="card in filteredCards"
+      :key="card.title"
+      cols="12"
+      sm="12"
+      md="12"
+      lg="6"
+      :xl="filteredCards.length > 3 ? 4 : 6"
+    >
```

Comment:
please avoid style updates in code spots that has no code changes as it's very confusing

Reply:
it's handling the card sizes across all screens